crates/sui-bridge/src/metrics.rs

```diff
@@ -15,6 +20,140 @@ const FINE_GRAINED_LATENCY_SEC_BUCKETS: &[f64] = &[
     200., 250., 300., 350., 400.,
 ];
 
+pub struct MetricsPushClient {
+    certificate: std::sync::Arc<sui_tls::SelfSignedCertificate>,
+    client: reqwest::Client,
+}
+
+impl MetricsPushClient {
+    pub fn new(metrics_key: sui_types::crypto::NetworkKeyPair) -> Self {
+        use fastcrypto::traits::KeyPair;
+        let certificate = std::sync::Arc::new(sui_tls::SelfSignedCertificate::new(
+            metrics_key.private(),
+            sui_tls::SUI_VALIDATOR_SERVER_NAME,
```

Comment:
should we use different values?

Reply:
Not unless we run a dedicated metrics proxy for the bridge: https://github.com/MystenLabs/sui/blob/d275762040247747ef429eda6309ffbef005c28b/crates/sui-proxy/src/admin.rs#L260-L263